### PR TITLE
Build dev doc

### PR DESCRIPTION
**Pull request by LysandreJik**

### CODE REVIEW

File: .github/workflows/build_dev_documentation.yml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      - name: Print
+        run: |
+          echo $EVENT_CONTEXT
+
+      - name: Comment PR
+        uses: thollander/actions-comment-pull-request@v1
+        if: github.event.action == 'opened' || github.event.action == 'reopened'
+
+        with:
+          message: 'Thank you for your PR! The docs for this PR live [here](https://moon-ci-docs.huggingface.co/docs/transformers/pr_15110).\n\nAll of your documentation changes will be reflected on that endpoint.'
```
Comment (sgugger) [inline .github/workflows/build_dev_documentation.yml:42]: Should be PR number here, not 15110.

File: .github/workflows/delete_dev_documentation.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    env:
+      PR_NUMBER: ${{ github.event.number }}
+      ACTION: ${{ github.event.pull_request.action }}
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: 'huggingface/doc-build-dev'
+          path: doc-build-dev
+          token: ${{ secrets.HUGGINGFACE_PUSH }}
```
Comment (sgugger) [inline .github/workflows/delete_dev_documentation.yml:23]: Same, this won't work for forks.

**Comment by LysandreJik** [comment]
I'll remove the `\n\n` but other than that, does the message look good to you?

**Comment by LysandreJik** [comment]
Closing

**Pull request closed without merging**